examples/ios/objc/Draw/Models/SwatchColor.m

```diff
@@ -0,0 +1,148 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2014 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "SwatchColor.h"
+
+@implementation SwatchColor
+
++ (NSDictionary *)sharedColors
+{
+    static NSDictionary<NSString *, UIColor*> *swatchColors;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
```

Comment:
Wouldn't it be simpler to store the colors in a static `NSDictionary`, get rid of the `SwatchColor` class, and just get colors by subscripting from the dictionary? Seems like it'd cut down on a lot of code and would make things simpler and faster by reducing runtime allocations.